csci3230-project/server.js

```diff
@@ -1,14 +1,15 @@
 var express = require('express');
 var exec = require('child_process').exec;
 var bodyParser = require('body-parser');
+var imgur = require('imgur'); //TODO: npm instal --save imgur
```

Comment:
The `package.json` of the project is still in the project root folder, can you move it to this directory and run the command you have in the `//TODO` comment? to ensure the pckage.json has the imgur library as a dependancy